.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,62 @@
+name: Docker - Build Jammy/Humble Essentials
+
+on:
+  pull_request:
+    paths:
+      - "devcontainer/**"
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: bdaiinstitute/spot_ros2_jammy_humble
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Workaround: https://github.com/docker/build-push-action/issues/461
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@79abd3f86f79a9d68a23c75a09a9a85889262adf
```

Comment:
Is it the standard practice to use actions versioned by their sha, as opposed by their version numbers?

Reply:
I have no idea. I've seen both.